setup.py

```diff
@@ -18,14 +18,15 @@
 install_requires = [
     "prettytable",
     "ipython>=1.0",
-    "sqlalchemy<2",
+    "sqlalchemy<=1.4.46",
     "sqlparse",
     "ipython-genutils>=0.1.0",
     "sqlglot",
     "jinja2",
     "sqlglot>=11.3.7",
     "ploomber-core>=0.2.4",
-    'importlib-metadata;python_version<"3.8"'
+    "ipywidgets",
```

Comment:
this will make ipywidgets a required dependency, let's make it optional by calling [`check_installed`](https://ploomber-core.readthedocs.io/en/latest/dependencies.html#check-installed) if we detect the user passed `--interactive`

Let's include this in the tutorial so they know they need it, something like "ipywidget is required to use this feature, install it with pip install ..."

Reply:
note that I just added `check_installed` to ploomber core so we need to pin it in setup.py `ploomber-core>=0.2.7`

Reply:
Handled in [e6b45dd](https://github.com/ploomber/jupysql/pull/293/commits/e6b45dd39b8b01856b8351407f346fe2ad583668)